org.eclipse.m2e.mavenarchiver/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,15 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=11
```

Comment:
Has to be set to Java-17. Ideally you just set it via the Right-Click -> Build Path -> Configure Build Path and then set the JRE System Library. This also updates some other settings.